src/main/scala/com/campudus/tableaux/database/Tableaux.scala

```diff
@@ -285,10 +118,10 @@ class Tableaux(verticle: Verticle) {
   } yield Cell(linkColumn, rowId, Link(List((value._2, v))))
 
   def getAllTables(): Future[TableSeq] = for {
-    seqOfTableInformation <- tableStruc.getAll()
+    seqOfTableInformation <- tableStruc.getAll
   } yield {
-    TableSeq(seqOfTableInformation map { case (id, name) => Table(id, name) })
-  }
+      TableSeq(seqOfTableInformation map { case (id, name) => Table(id, name) })
+    }
```

Comment:
Fix intendation please:
```
for {
  x <- something
} yield {
  somethingElse()
}
```

`yield` part should get the same intendation like the `for` part.